src/options.ts

```diff
@@ -133,13 +155,23 @@ export function _setDefaultOptions(options: Partial<Options> = {}): Options {
     logInjectionEnabled: options.logInjectionEnabled,
     instrumentations: options.instrumentations,
     tracerConfig: tracerConfig,
+    tracesExporter: options.tracesExporter,
     spanExporterFactory: options.spanExporterFactory,
     spanProcessorFactory: options.spanProcessorFactory,
     propagatorFactory: options.propagatorFactory,
   };
 }
 
-export function defaultSpanExporterFactory(options: Options): JaegerExporter {
+export function defaultSpanExporterFactory(options: Options): SpanExporter {
```

Comment:
I think we should call this `otlpSpanExporterFactory` instead of default. The constructor decides what is default and what isn't. Factory itself has no business knowing if it is used as the default one.